praw/models/reddit/redditor.py

```diff
@@ -293,6 +295,22 @@ def moderated(self) -> List["praw.models.Subreddit"]:
                 print(subreddit.display_name)
                 print(subreddit.title)
 
+        .. note::
+
+            A ``prawcore.ServerError`` may be raised if the redditor moderates a large
+            number of subreddits. If that happens, try switching to :ref:`read-only mode
+            <read_only_application>`. For example,
```

Comment:
```suggestion
            number of subreddits (>3000). If that happens, try switching to
            :ref:`read-only mode <read_only_application>`. For example,
```
Could we define large here?

Reply:
I guess we could discuss this in a separate PR, but I deliberately made this line vague because the cutoff is out of PRAW's control, and could change over time. The endpoint isn't even documented.

Reply:
Ah okay makes sense, we can leave it as is then.